documentation/intro_forsc.md

```diff
@@ -36,10 +36,13 @@ See {ref}`server-and-env-options` for more information.
 
 ### Storage Resources
 
-The Fornax Science Console offers two types of storage:
+The Fornax Science Console currently offers 3 types of storage:
 
--   **Persistent home directory:** Each user has a 10 GB home directory on the file system for storing important files that need to be saved long term.
--   **Extended temporary storage:** For project work lasting several months, users can request up to 5 TB of object storage on {term}`S3`, intended for data that doesn’t require fast file system access and won’t be stored permanently.
+-   **Persistent home directory:** Each user has a 20 GB private home directory on the file system for storing important files that need to be saved long term. Users can request a size increase incremently up to 100 GB.
```

Comment:
We should tell people how to request the increase. I think the answer right now is to open a Support topic in discourse. Instead of adding yet another external link to that, it probably makes more sense to add an internal link to the Fornax Community Forum page in these docs.